pkg/services/container/announcement/load/storage/storage.go

```diff
@@ -40,21 +40,28 @@ type storageKey struct {
 type Storage struct {
 	mtx sync.RWMutex
 
-	mItems map[storageKey]*usedSpaceEstimations
+	estLifeCycle uint64
+	mItems       map[storageKey]*usedSpaceEstimations
 }
 
 // Prm groups the required parameters of the Storage's constructor.
 //
 // The component is not parameterizable at the moment.
-type Prm struct{}
+type Prm struct {
+	// EstimationsLifeCycle is a number (in epochs) estimations
```

Comment:
```suggestion
	// EstimationsLifeCycle is a longevity (in epochs) of estimations
```
? otherwise w/o `(in epochs)` sentence seems weird to me